### ISSUE TITLE
Post-install message about Ruby Sass end-of-life

### ISSUE THREAD

**Issue by uchilaka** [rails/sass-rails#435]
I got this Post-install message when creating a new rails app: 

```bash
Post-install message from sass:

Ruby Sass has reached end-of-life and should no longer be used.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  https://sass-lang.com/blog/posts/7828841
```

I checked my project `Gemfile` and the only sass dependency we've got is on `sass-rails`. Is there a change that needs to be made to fix the source of this warning? Perhaps a switch to using the `sassc` gem instead? (New to rails so haven't figured out yet how that might impact `sass-rails` dependency on `sass`)

**Comment by uchilaka** [comment]
I found a dependency definition on `sassc` in this file: https://github.com/rails/sass-rails/blob/master/lib/sass-rails.rb on the `master` branch. Does that mean this is fixed / pending release or a non-issue in some other way?

**Comment by uchilaka** [comment]
Perhaps the change needed is to the default gem dependency definitions for rails `6.0.0`. Here's what came through for my newly created rails app:

```gemfile
...
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5'
...
```

**Comment by rafaelfranca** [comment]
It is fixed on the latest release 6.0.0
